conda-forge.yml

```
@@ -1,3 +1,4 @@
 build_platform: {osx_arm64: osx_64}
 conda_forge_output_validation: true
 test_on_native_only: true
+provider: {linux_aarch64: default}
```

```suggestion
provider: {linux_aarch64: default, linux_ppc64le: default}}
```

Why not just add the end package you want to the migrator?
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/arch_rebuild.txt

---

Hey, I'm really new to the conda-forge ecosystem and I didn't know about this file. I will do a PR on this repo then, thank you!

---

we can push this through of you want. if it is only one package then this is a fine approach.

we are trying to keep ppcl64 and aarch in sync. so I'm requesting that you make the PR for both architectures at once.

---

Oh, I'm trying to add the support for `aarch64` on a lot of packages. Can I just add the top packages in the `arch_rebuild.txt` file and it will build all the dependencies for `aarch64` automatically?

---

yes.

---

Oh ok, thank you very much for the help!